apps/i18n/netsim/en_us.json

```diff
@@ -30,6 +32,8 @@
   "sendAMessage": "Send a Message",
   "sentMessageLog": "Sent Message Log",
   "to": "To",
+  "unlimited": "Unlimited",
   "workspaceHeader": "Internet Simulator",
+  "xBitsPerPacket": "{x} bits per packet",
```

Comment:
numBitsPerPacket might be marginally easier on translators. Opinions, @tanyaparker ?